bots/images/scripts/ovirt.setup

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+set -eux
+
+echo foobar | passwd --stdin root
+
+/usr/bin/systemctl stop firewalld NetworkManager kdump
+/usr/bin/systemctl disable firewalld NetworkManager kdump
+iptables -F
+
+FQDN=ovirt.cockpit.lan
+hostnamectl set-hostname ${FQDN}
+
+# Latest stable repos
+rpm -ivh http://resources.ovirt.org/pub/yum-repo/ovirt-release41.rpm
```

Comment:
Unfortunately these are the official ovirt install repos, looking at 
http://resources.ovirt.org/pub/yum-repo/ there are only released versions and nightly build, which I wouldn't recommend to use.
A single release typically covers 3/4-1 year